www/src/Ecwec/Bundle/UserBundle/DataFixtures/ORM/LoadUserData.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+
+namespace Ecwec\Bundle\UserBundle\DataFixtures\ORM;
+
+use Doctrine\Common\DataFixtures\FixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Ecwec\Bundle\UserBundle\Entity\User;
+
+class LoadUserData implements FixtureInterface, ContainerAwareInterface
+{
+    /**
+     * @var ContainerInterface
+     */
+    private $container;
+
+    /**
+     * {@inheritDoc}
+     */
+    public function setContainer(ContainerInterface $container = null)
+    {
+        $this->container = $container;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function load(ObjectManager $manager)
+    {
+        $userManager = $this->container->get('fos_user.user_manager');
+
+        // Create our user and set details
+        /** @var User $user */
+        $user = $userManager->createUser();
```

Comment:
Can we use this project? [Alice](https://github.com/nelmio/alice). It would seem more practical to write these in yml. And if we ever wanted to get crazy, we could incorporate the [Faker](https://github.com/fzaninotto/Faker) library to generate lots of demo data.